packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -0,0 +1,271 @@
+import '@aws-cdk/assert-internal/jest';
+import { AccountRootPrincipal, Group, Role, User } from '@aws-cdk/aws-iam';
+import { App, Stack, Tags } from '@aws-cdk/core';
+import { AcceptLanguage, Portfolio } from '../lib';
+
+describe('Portfolio', () => {
+
+  test('default portfolio creation', () => {
+    const app = new App();
+    const stack = new Stack(app);
+
+    new Portfolio(stack, 'MyPortfolio', {
+      name: 'testPortfolio',
+      provider: 'testProvider',
+    });
+
+    expect(stack).toMatchTemplate({
+      Resources: {
+        MyPortfolio59CCA9C9: {
+          Type: 'AWS::ServiceCatalog::Portfolio',
+          Properties: {
+            DisplayName: 'testPortfolio',
+            ProviderName: 'testProvider',
+          },
+        },
+      },
+    });
+  }),
+
+  test('portfolio with explicit acceptLanguage and description', () => {
+    const app = new App();
+    const stack = new Stack(app);
+
+    new Portfolio(stack, 'MyPortfolio', {
+      name: 'testPortfolio',
+      provider: 'testProvider',
+      description: 'test portfolio description',
+      acceptLanguage: AcceptLanguage.ZH,
+    });
+
+    expect(stack).toMatchTemplate({
```

Comment:
Can you use `toHaveResourceLike()` instead? This test repeats the same assertion on `DisplayName` and `ProviderName` from the previous test. They should only be checking for `Description` and `AcceptLanguage`.

Same comment in the tests below that use `toMatchTemplate()`.